src/content/en/2020/jamstack.md

```diff
@@ -3,21 +3,265 @@
 discuss: 2053
 results: https://docs.google.com/spreadsheets/d/1BCC5Q4tePpTl8TiaGmSxBc9Lh2to7xBfVPMULFOBwvk/
 queries: 17_Jamstack
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
-unedited: true
+featured_quote: Stats suggest almost twice as many Jamstack sites exist now than in 2019. Developers enjoy a better development experience but that doesn't always translate to a better user experience. Let’s work towards continued progress for improving the real-user experience of browsing Jamstack sites.
+featured_stat_1: 154%
+featured_stat_label_1: more mobile websites built with Jamstack in 2020
+featured_stat_2: 1 gram
+featured_stat_label_2: CO2 is emitted by the median Jamstack page load
+featured_stat_3: 38%
+featured_stat_label_3: Jamstack sites are built using Next.js (three times more than any other framework!)
 ---
+
+## Introduction
+
+Jamstack is a relatively new concept of an architecture designed to make the web faster, more secure, and easier to scale. It builds on many of the tools & workflows which developers love and which bring maximum productivity.
+
+The core principles of Jamstack are pre-rendering your site pages and decoupling the frontend from the backend. It relies on the idea of delivering the frontend content hosted separately on a CDN provider that uses APIs (for example, a headless CMS) as its backend if any.
+
+The [HTTP Archive][1] crawls [millions of pages][2] every month and runs them through a private instance of [WebPageTest][3] to store key information on every page. (You can learn more about this in our [methodology][4]). In the context of Jamstack, HTTP Archive provides extensive information on the usage of the frameworks and CDNs for the entire web. This chapter consolidates and analyzes many of these trends.
+
+### Goals: Growth of Jamstack & Real User Perf
+
+The goals of this chapter are to estimate and analyze the growth of the Jamstack sites, the performance of popular Jamstack frameworks, as well as an analysis of real user experience using the Core Web Vitals metadata.
+
+## Adoption: What is the overall level of Jamstack adoption amongst all websites, across desktop and mobile?
```

Comment:
This is VERY long. Look at what it looks like in the table of contents:

![Jamstack ToC](https://user-images.githubusercontent.com/10931297/101237719-f4060780-36d2-11eb-80d5-865adb5cd0ac.png)

```suggestion
## Adoption of Jamstack adoption
```